lms/djangoapps/courseware/date_summary.py

```diff
@@ -204,10 +204,20 @@ class CertificateAvailableDate(DateSummary):
     title = ugettext_lazy('Certificate Available')
 
     @property
+    def active_certificates(self):
+        return [
+            certificate for certificate in self.course.certificates.get('certificates', [])
+            if certificate.get('is_active', False)
+        ]
+
+    @property
     def is_enabled(self):
         return (
-            self.date is not None and datetime.datetime.now(utc) <= self.date and not self.course.self_paced and
-            waffle.waffle().is_enabled(waffle.INSTRUCTOR_PACED_ONLY)
+            self.date is not None
```

Comment:
You could do the all() thing that Alex did in one of his PRs a few days ago.

`return all([self.date is not None, 
datetime.datetime.now(utc) <= self.date, 
not self.course.self_paced,
waffle.waffle().is_enabled(waffle.INSTRUCTOR_PACED_ONLY),
len(self.active_certificates) > 0])`

Reply:
Or, alternatively, putting the "and"s at the end:

```
return (
   self.date is not None and
   datetime. datetime.now(utc) <= self.date and
   not self.course.self_paced and 
   waffle.waffle().is_enabled(waffle.INSTRUCTOR_PACED_ONLY) and 
   len(self.active_certificates) > 0
```

Reply:
💯

Reply:
@ssemenova I like the 'all' thing, I think that's cool, it doesn't protect against the `self.date is not None` early return though -- so I'll move the `and`.